doc/source/index.rst

```diff
@@ -138,6 +138,7 @@ Management Improvements
    management/start_transaction_with_consistent_snapshot
    management/super_read_only
    management/extended_show_grants
+   management/ssl-improvement
```

Comment:
standartize on hyphen or underscore as word separator

Reply:
there is no standard in terms of file names